File: exercicios/1159/1159.py
while True:
    i = 1
    soma = 0
    n = int(input())
    if n == 0:
        break
    if n != 0:
        while i <= 5:
            if n % 2 == 0:
                soma = soma + n
                n = n + 1
                i = i+ 1
            else:
                n = n + 1

    
        print(soma)    
    

File: exercicios/JavaScript/1159/1159.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

while(true){
    var n = parseInt(lines.shift()), soma = 0;

    if(n == 0){
        break;
    }else{
        for(let i=n; i<n+10; i++){
            if(i % 2 == 0){
                soma += i;
            }
        }
        console.log(soma)
    }
}